### PR TITLE
[Bug fix] Fix seek by time error

### LINKED ISSUE #194: test case for consumer SeekByTime
Hi, 
Is it testify enough to not check published time for message in the test case for consumer [SeekByTime](https://github.com/apache/pulsar-client-go/blob/fc390a6a37f3cbd94ac46b3b5e4239b3ca5df875/pulsar/consumer_test.go#L823) ?
[test assert here](https://github.com/apache/pulsar-client-go/blob/fc390a6a37f3cbd94ac46b3b5e4239b3ca5df875/pulsar/consumer_test.go#L873) 
any idea?

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

Fixes #194 

### Modifications

- Fix seek by time

### CODE REVIEW

**Comment by mileschao** [comment]
@merlimat @sijie PR merge wanted. Thx :-)

File: pulsar/consumer_partition.go [hunk 1]
```diff
@@ -323,7 +323,7 @@ func (pc *partitionConsumer) internalSeekByTime(seek *seekByTimeRequest) {
 	cmdSeek := &pb.CommandSeek{
 		ConsumerId:         proto.Uint64(pc.consumerID),
 		RequestId:          proto.Uint64(requestID),
-		MessagePublishTime: proto.Uint64(uint64(seek.publishTime.Unix())),
+		MessagePublishTime: proto.Uint64(uint64(seek.publishTime.UnixNano() / 1e6)),
```
Comment (cckellogg) [inline pulsar/consumer_partition.go]: Should we use a constant? `seek.publishTime.UnixNano() / int64(time.Millisecond)`

Comment (wolfstudy) [reply pulsar/consumer_partition.go]: Thanks @cckellogg will fix it

**Comment by wolfstudy** [comment]
@cckellogg PTAL thanks.

**Pull request merged**